Assets/UnityShell/Editor/Scripts/UnityShellEditorWindow.cs

```diff
@@ -12,15 +12,24 @@ static class Styles
 		{
 			public static readonly GUIStyle textAreaStyle;
 
+			// Default background Color(0.76f, 0.76f, 0.76f)
+			private static readonly Color BgColorLightSkin = new Color(0.87f, 0.87f, 0.87f);

```

Comment:
Not so important thing: I'd start with lower case here (just as "textAreaStyle" above).

Reply:
Yep, should stay consistent, fixed.

Reply:
@marijnz I noticed that that's for a public value, a private value is a few lines below and starts with a "_nameHere" format

Reply:
I guess that's mostly since there is a public variable with the same name and the private variable is just a helper that should never be accessed directly.
Sadly I can't find any private static readonly variable, @marijnz is there any naming convention documentation of the one you're using (something like https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions)?

Reply:
I mean, if anything it would work for now. It's not that deep. But IMO the public variables should start with a capital, but that's just me.

Reply:
Hi, @wolfreak99, good point. @JohannesDeml is right, I'm using the underscore there to mark it as a "shadow variable". As for a naming conventions, it's not something I wrote down somewhere. I'm indifferent to it, besides that I like consistency, for that reason I'm generally going with what Unity is going (for better or worse ;))! And I just checked a couple of their classes and their public statics are indeed camelCase.

Anyway, it's indeed not so important, and this PR could've of course already been merged, so doing that now. But feel free to still reply here